projectfiles/Xcode/README.md

```diff
@@ -1,4 +1,5 @@
 # Building with Xcode
+This README describes the way, how wesnoth release packages are build for macOS.
```

Comment:
"This README describes the way **to create the** wesnoth release packages for macOS."